configure.d/1_bd_part_count.conf

```diff
@@ -9,7 +9,11 @@
 check() {
 	cur_name=$(basename $2)
 	config_file_path=$1
-	if compile_module $cur_name "struct gendisk *disk = NULL; struct xarray xa; xa = disk->part_tbl;" "linux/genhd.h"
+	if compile_module $cur_name "struct gendisk *disk = NULL; struct xarray xa; xa = disk->part_tbl;" "linux/blkdev.h"
+	then
+		echo $cur_name "1" >> $config_file_path
+
```

Comment:
looks like it can be combined into a single if with logical "or" in condition, just like in 1_bd_first_part.conf